packages/contracts-bedrock/deploy-config/mainnet.json

```diff
@@ -1,23 +1,23 @@
 {
   "finalSystemOwner": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A",
-  "controller": "0x78339d822c23d943e4a2d4c3dd5408f66e6d662d",
-  "portalGuardian": "0x78339d822c23d943e4a2d4c3dd5408f66e6d662d",
-  "l1StartingBlockTag": "0x126e52a0cc0ae18948f567ee9443f4a8f0db67c437706e35baee424eb314a0d0",
+  "controller": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A",
+  "portalGuardian": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A",
+  "l1StartingBlockTag": "",
   "l1ChainID": 1,
   "l2ChainID": 10,
   "l2BlockTime": 2,
   "maxSequencerDrift": 600,
   "sequencerWindowSize": 3600,
   "channelTimeout": 300,
-  "p2pSequencerAddress": "0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65",
+  "p2pSequencerAddress": "0xAAAA45d9549EDA09E70937013520214382Ffc4A2",
   "batchInboxAddress": "0xff00000000000000000000000000000000000010",
   "batchSenderAddress": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
```

Comment:
Shouldn't this be: https://etherscan.io/address/0x6887246668a3b87f54deb3b94ba47a6f63f32985